manim/mobject/opengl/opengl_mobject.py

```diff
@@ -2348,11 +2348,14 @@ def construct(self):
                 continue
 
             if key in ("points", "bounding_box"):
-                func = path_func
+                self.data[key] = path_func(
+                    mobject1.data[key], mobject2.data[key], alpha
+                )
             else:
-                func = interpolate
+                self.data[key][:] = interpolate(
+                    mobject1.data[key], mobject2.data[key], alpha
+                )
 
-            self.data[key][:] = func(mobject1.data[key], mobject2.data[key], alpha)
```

Comment:
Do you, by any chance, have an explanation why the assignment has to be different in the two cases?

Reply:
https://github.com/ManimCommunity/manim/blob/e7b9cf764b446e69ea291d70e7f30522beb76ec6/manim/mobject/opengl/opengl_vectorized_mobject.py#L1330-L1341

It is related to this. Because we are copying the curves that are inside the a,b range and shortening the two at the border. This means we potentially have less curves than before. So it's not a perfectly smooth interpolation. For that we would need the Bezier remapping function which then replaces
https://github.com/ManimCommunity/manim/blob/e7b9cf764b446e69ea291d70e7f30522beb76ec6/manim/mobject/opengl/opengl_vectorized_mobject.py#L1335-L1336
to be something like
```py
points = bezier_triplets[lower_index + 1 : upper_index]
smooth_bezier_remap(old_points = points, map_to_n_curves = num_quadratics-2)
-> returns a list of length `num_quadratics-2` containing the new bezier points.
```

Reply:
I added a TODO in the code

Reply:
This is now reverted